### DISCUSSION TITLE
Question about HamburgerMenu

### DISCUSSION THREAD

**Discussion by markusstephany** [MahApps/MahApps.Metro#4074] [Q&A] [answered]
Hi there,

I want to use the HamburgerMenu but not its content. That is I want to use ICommands in the menu items and the underlying ViewModel to perform certain commands (e.g. navigating the main part of the window to a different view) I am using Prism's Regionmanager for views navigation, and thus I do not want to put the view(model)s in the HamburgerMenu's Content.  

The problem is that I cannot manage to get rid of the (now empty) content part of the HamburgerMenu in my window. If it is expanded, it shows the command labels, but if it is collapsed, there is just an empty space. I am getting nuts, but I find no property or whatever to set the content part width to 0 without ruining the expansion/collapsing of the HamburgerMenu.

![HamburgerMenu](https://user-images.githubusercontent.com/51796015/112201695-cce72d80-8c10-11eb-8736-b87532f172af.png)

Any advice?
TIA and BR
Markus

**Comment by timunie** [comment, marked as answer]
Hi @markusstephany 

I still don't get your point why you don't want to use the `HamburgerMenu.Content` to hold your content. It can by anything like any other `ContentControl` so there is no need to use the items content. 

If this is not what you like to do you can also use the `SplitView` to build your own `HamburgerMenu`-like control. I have an example here: 
https://github.com/MahApps/MahApps.Metro.IconPacks/pull/232#issuecomment-743749756 

![image](https://user-images.githubusercontent.com/47110241/113035523-9a4eaf00-9193-11eb-8f0e-b4e45ee9c34a.png)

Happy coding
Tim

**Reply by markusstephany** [reply]
"Tomaten auf den Augen"... Thanks for pointing me to the right direction. I did just not get the fact that I can use HamburgerMenu like any other ContentControl. Works like a charm :-D
